components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -3153,7 +3188,7 @@ subroutine ice_deposition_sublimation(qitot_incld,nitot_incld,t,    &
    real(rtype), intent(in)  :: qv
    real(rtype), intent(out) :: qidep
    real(rtype), intent(out) :: qisub
-   real(rtype), intent(out) :: nisub
+   real(rtype), intent(out) :: nisub 
```

Comment:
trailing whitespace. What editor are you using to code? If you use emacs, I can provide some minor configuration changes so that trailing whitespaces are highlighted and easy to remove.